src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.cmd

```diff
@@ -103,7 +103,11 @@ Robocopy "%BUILDTOOLS_PACKAGE_DIR%\." "%TOOLRUNTIME_DIR%\." "MSBuild.runtimeconf
 
 :: Copy Portable Targets Over to ToolRuntime
 if not exist "%PACKAGES_DIR%\generated" mkdir "%PACKAGES_DIR%\generated"
-set PORTABLETARGETS_PROJECT=%PACKAGES_DIR%\generated\project.json
+if [%BUILDTOOLS_USE_CSPROJ%]==[] (
+  set PORTABLETARGETS_PROJECT=%PACKAGES_DIR%\generated\project.json
```

Comment:
Probably easy enough to just add another variable for the extension and use it for both projects.